maestro-ios-xctest-runner/maestro-driver-iosUITests/Routes/Models/LaunchAppRequest.swift

```diff
@@ -0,0 +1,5 @@
+struct LaunchAppRequest: Codable {
+    let bundleId: String
+    let arguments: [String]?
```

Comment:
Is this not passed as map here:

```
data class LaunchAppRequest(val appId: String, val launchArguments: Map<String, Any>, val maestroSessionId: UUID?)
```

How is this converted to array? Also not sure why we need environment here.

We just need support launch arguments with all type of values mentioned in docs here:

https://docs.maestro.dev/api-reference/commands/launchapp#sending-launch-arguments